localstack/services/sns/provider.py

```diff
@@ -169,9 +181,66 @@ def publish_message(
             message_attributes,
         )
     )
+
     return message_id
 
 
+def message_to_endpoint(
+    target_arn,
+    message,
+    structure,
+):
+    sns_client = aws_stack.connect_to_service("sns")
+    app_name = target_arn.split("/")[-2]
+    platform_name = target_arn.split("/")[-3]
+
+    try:
+        endpoint_attributes = sns_client.get_endpoint_attributes(EndpointArn=target_arn)[
+            "Attributes"
+        ]
+    except botocore.exceptions.ClientError:
+        LOG.warning(f"Missing attributes for endpoint: {target_arn}")
+        return
+
+    try:
+        platform_apps = sns_client.list_platform_applications()["PlatformApplications"]
+        app = [x for x in platform_apps if app_name in x["PlatformApplicationArn"]][0]
+    except IndexError:
+        LOG.warning(f"Missing Platform Application for endpoint: {target_arn}")
+        return
+
+    if structure == "json":
+        message = json.loads(message)
+
+    response = None
+    if platform_name == "GCM":
+        response = send_message_to_GCM(app["Attributes"], endpoint_attributes, message["GCM"])
+    # TODO: Add support for other platforms
```

Comment:
I'll be adding Apple's APN in the next few days, would love to add this for local testing of our iOS mobile app :)

Reply:
hi @AllenKaplan the issue and the PR focus on GCM, if you need another feature or problem fixed, please open another issue. With that, another programmer can take care of it if I'm not available or I'm unable to code it myself.